utils/zerovec/src/ule/macros.rs

```diff
@@ -0,0 +1,27 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+#[macro_export]
+macro_rules! impl_ule_from_array {
+    ($aligned:ty, $unaligned:ty, $single:path) => {
+        #[doc = concat!("Convert an array of `", stringify!($aligned), "` to an array of `", stringify!($unaligned), "`.")]
+        pub const fn from_array<const N: usize>(arr: [$aligned; N]) -> [Self; N] {
+            if N == 0 {
+                return unsafe { *(&arr as *const _ as *const [Self; N]) };
+            }
+            let mut result = [$single(arr[0]); N];
+            let mut i = 0;
```

Comment:
```suggestion
            let mut i = 1;
```

Reply:
I was contemplating doing the same, but ended up not changing anything since I'd assumed the compiler would figure it out anyway and "starting" at 1 is slightly confusing. 

I added a comment and applied your change